.github/workflows/pypi-publish.yaml

```diff
@@ -0,0 +1,49 @@
+on:
+  workflow_dispatch:
+  release:
+    types:
+      - released
+      - prereleased
```

Comment:
Note that this section differs from [the corresponding workflow in assesspy](https://github.com/ccao-data/assesspy/blob/488cd1e6f0da424136f18c6673b1495aa7a8f8a5/.github/workflows/pypi-publish.yaml#L4), in order to support pushing to the PyPI test instance on prereleases, which assesspy is not currently configured to do.